client/src/components/MapElement/MapElement.js

```diff
@@ -11,6 +18,14 @@ MapElement.propTypes = {
 export function MapElement({ className }) {
```

Comment:
see GitHub actions, className is not used. please directly embed it in your top level return's className string